features/extension-mgt/org.wso2.carbon.identity.extension.mgt.feature/resources/extensions/connections/custom-authentication/info.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "custom-authentication",
+  "name": "Custom Authentication",
+  "description": "Enable login for users with external authentication service.",
+  "image": "",
+  "category": "DEFAULT",
+  "displayOrder": -1,
```

Comment:
What is meant by -1. What is purpose of this parameter

Reply:
This is to represent the order in which the card will be displayed on the connections page. -1 implied that this will not be displayed and an updated grouped template will be displayed instead.